x/emissions/metrics/metrics_util.go

```diff
@@ -29,10 +29,10 @@ func IncrementRpcRequestCounter(endpoint string, err *error) {
 // Measures the RPC request latency in milliseconds
 // Metric Name:
 //
-//	allora_rpc_request_latency_ms
+//	allora_request_latency_ms
 func MeasureRpcRequestLatency(endpoint string, startTime time.Time) {
 	metrics.MeasureSinceWithLabels(
-		[]string{"allora", "request", "latency_ms"},
+		[]string{"allora", "emissions", "request", "latency_ms"},
```

Comment:
```suggestion
		[]string{"allora", "emissions", "request", "duration_ms"},
```

I think `duration` would be more appropriate as latency suggests a delay